packages/ember-glimmer/tests/integration/syntax/if-unless-test.js

```diff
@@ -142,4 +405,79 @@ moduleFor('Syntax test: {{#unless}}', class extends RenderingTest {
     this.assertText('T1F2');
   }
 
-});
+}
+
+moduleFor('Syntax test: {{#if}}', class extends SharedConditionalsTest {
+
+  templateFor({ cond, truthy, falsy }) {
+    return `{{#if ${cond}}}${truthy}{{else}}${falsy}{{/if}}`;
+  }
+
+  ['@test it renders and hides the given block based on the conditional']() {
+    this.render(`{{#if cond1}}T1{{/if}}{{#if cond2}}T2{{/if}}`, { cond1: true, cond2: false });
+
+    this.assertText('T1');
+
+    this.inZone(() => this.rerender());
+
+    this.assertText('T1');
+
+    this.inZone(() => set(this.context, 'cond1', false));
+
+    this.assertText('');
+
+    this.inZone(() => {
+      set(this.context, 'cond1', true);
+      set(this.context, 'cond2', true);
+    });
+
+    this.assertText('T1T2');
+
+    this.inZone(() => {
+      set(this.context, 'cond1', true);
+      set(this.context, 'cond2', false);
+    });
+
+    this.assertText('T1');
+  }
+
+}, BASIC_TRUTHY_TESTS, BASIC_FALSY_TESTS);
+
+moduleFor('Syntax test: {{#unless}}', class extends SharedConditionalsTest {
+
+  templateFor({ cond, truthy, falsy }) {
+    return `{{#unless ${cond}}}${falsy}{{else}}${truthy}{{/unless}}`;
+  }
+
+  ['@test it renders and hides the given block based on the conditional']() {
```

Comment:
This is the only test for without inverse (primarily because I haven't figured out how to abstract it yet)